packages/bezier-react/src/components/Overlay/Overlay.tsx

```diff
@@ -230,11 +232,14 @@ export const Overlay = forwardRef<HTMLDivElement, OverlayProps>(
       return null
     }
 
+    const zIndexClassName = getZIndexClassName(zIndex)
+
     const Content = (
       <ThemeProvider themeName={themeName}>
         <div
           className={classNames(
             styles.Overlay,
+            zIndexClassName,
```

Comment:
```suggestion
            getZIndexClassName(zIndex)
```

inline으로 해도 될 것 같습니다

Reply:
넵 알겠습니다!